cabot/metricsapp/models/elastic.py

```diff
@@ -194,17 +194,16 @@ def _parse_series(self, series, series_name=None):
         :return: (series, (timestamp, datapoint)) pairs
         """
 
-        islist = isinstance(series, list)
+        this_series_name = series_name
```

Comment:
supernit: can we remove this and change series_name in line 205 to subseries_name? I don't really like this_series_name, it's not very specific